wordpress-mobile/WordPress-FluxC-Android
# Woo: Add a missing Coupon property

This PR adds a missing Coupon `amount` field to the DTO and and entity.

**To test:**
1. Run the DB migration and make sure it passes.
2. Run the example and verify there is a new `amount` column in the DB, using the App inspection tool
